layout.go

```diff
@@ -8,4 +8,10 @@ type Layout interface {
 	// MinSize calculates the smallest size that will fit the listed
 	// CanvasObjects using this Layout algorithm.
 	MinSize(objects []CanvasObject) Size
+	// GetPaddings returns the top, bottom, left and right paddings used
+	// by this layout.
+	GetPaddings() (float32, float32, float32, float32)
```

Comment:
Just FYI such an addition to a published interface is a breaking change.

Anyhow this topic has been discussed a lot and you can see replacement PRs coming in that take a less-sweeping change that can be adopted more easily.
Hopefully this resolves the use-case we missed?